python/ray/experimental/channel/torch_tensor_nccl_channel.py

```diff
@@ -1,45 +1,56 @@
 import io
 import logging
 import uuid
+from dataclasses import dataclass
 from types import ModuleType
-from typing import TYPE_CHECKING, Any, Callable, List, Optional, Tuple, Union
+from typing import TYPE_CHECKING, Any, List, Optional, Tuple, Union
 
 import ray
 import ray.util.serialization
 from ray.experimental.channel import ChannelContext
 from ray.experimental.channel.common import ChannelInterface
 from ray.experimental.channel.nccl_group import _NcclGroup
 from ray.experimental.channel.shared_memory_channel import SharedMemoryType
-from ray.experimental.channel.torch_tensor_type import TENSOR_METADATA_SIZE_BYTES
 from ray.util.annotations import DeveloperAPI
 
 if TYPE_CHECKING:
     import torch
 
     from ray.experimental.channel.shared_memory_channel import Channel
-    from ray.experimental.channel.torch_tensor_type import TorchTensorType
 
 
 # Logger for this module. It should be configured at the entry point
 # into the program using Ray. Ray provides a default configuration at
 # entry/init points.
 logger = logging.getLogger(__name__)
 
+# 100KB to store metadata and/or exceptions.
```

Comment:
100 kB

Reply:
Hmm I believe elsewhere we use KB instead of kB.